content/realtime/channel-params.textile

```diff
@@ -0,0 +1,91 @@
+---
+title: Channel Parameters
+section: realtime
+index: 32
+jump_to:
+  Help with:
+    - Overview#overview
+    - Using params with the Ably library#using-params-with-library
+    - Using params with non-Ably transports#using-params-with-other-transports
+  rewind parameter#rewind
+    - Examples#examples
+---
+
+h2(#overview). Overview
+
+Channel parameters are a general mechanism by which a client can express properties of a channel, or of its attachment to a channel. A set of channel parameters is simply a set of key/value pairs; the keys correspond to specific features that Ably defines. Having a general mechanism for specifying such parameters allows the feature set to be expanded over time without requiring any API change and, in most cases, without requiring any additional library functionality.
```

Comment:
This is written from the perspective of us, not the developer.  They don't care if it's extensible or not, they just want to know how to use channel params.  Anyway, this is very useful and I think whoever picks this up and figures out how it applies to the docs should worry about this point.

Reply:
I think we should have a section very early on which states supported channel params, with perhaps a list of upcoming (alpha) params too alluding to deltas for example.